tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/tests/AbstractPitclipseSWTBotTest.java

```diff
@@ -307,6 +312,32 @@ protected static void runProjectTest(final String projectName) throws CoreExcept
         new PitclipseSteps().runProjectTest(projectName);
     }
 
+    /**
+     * Asserts that the only active mutator was the given mutator.
+     * @param mutators which should be the only active mutator
+     */
+    protected static void mutatorIs(Mutators mutator) {
+        mutatorsAre(Arrays.asList(new Mutators[] { mutator }));
+    }
+
+    /**
+     * Asserts that the only active mutators are the given mutators.
+     * @param mutators which should be the only active mutators
+     */
+    protected static void mutatorsAre(Collection<Mutators> mutators) {
+        final String consoleText = PAGES.getConsole().getText();
+        // build String to match against console text
+        Iterator<Mutators> iterator = mutators.iterator();
+        StringBuilder sb = new StringBuilder();
+        while (iterator.hasNext()) {
+            sb.append(iterator.next().name());
+            if (iterator.hasNext()) {
+                sb.append(',');
+            }
+        }
+        assertThat(consoleText, containsString(String.format("mutators=[%s]", sb.toString())));
```

Comment:
Maybe I'm missing something, but since you want to check the exact mutators, why do you only check that the string is contained and not exact?

Reply:
I want to check inside the complete console text and the closing bracket makes this match exact mutators.

Reply:
So why not simply use JUnit `assertEquals`? Since you want to check the complete console text, `assertEquals` provides better tooling in Eclipse

Reply:
I don't want to check the complete console String. I want only to see if the mutators are contained inside console text in the exact way we expect them to be.